linux_os/guide/system/accounts/accounts-session/accounts_users_netrc_file_permissions/oval/shared.xml

```diff
@@ -0,0 +1,45 @@
+<def-group>
+  <definition class="compliance" id="{{{ rule_id }}}" version="1">
+    {{{ oval_metadata("Netrc User File In The Home Directory Must Not be group or world accessible") }}}
+    <criteria>
+      <criterion test_ref="test_accounts_users_home_netrc_file_permissions"
+                 comment="Netrc User File In The Home Directory Must Not be group or world accessible"/>
+    </criteria>
+  </definition>
+
+  <unix:password_object id="object_accounts_users_home_netrc_file_permissions_objects" version="1">
+    <unix:username datatype="string" operation="pattern match">.*</unix:username>
+    <filter action="include">state_accounts_users_home_netrc_file_permissions_interactive_gids</filter>
+  </unix:password_object>
+
+  <unix:password_state id="state_accounts_users_home_netrc_file_permissions_interactive_gids" version="1">
+    <unix:user_id datatype="int" operation="greater than or equal">{{{ gid_min }}}</unix:user_id>
```

Comment:
```suggestion
    <unix:user_id datatype="int" operation="greater than or equal">{{{ uid_min }}}</unix:user_id>
```

Reply:
Please, consider to use the [create_interactive_users_list_object](https://github.com/ComplianceAsCode/content/blob/master/shared/macros/10-oval.jinja#L980) macro to identify interactive users.

Here is an example: https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/accounts/accounts-session/accounts_user_dot_user_ownership/oval/shared.xml#L11